applications/nrf_desktop/configuration/nrf52833dk_nrf52833/prj_shell.conf

```diff
@@ -96,25 +95,10 @@ CONFIG_BT_HOGP_REPORTS_MAX=12
 CONFIG_ASSERT=y
 CONFIG_ASSERT_LEVEL=2
 
-CONFIG_LOG=y
-CONFIG_LOG_DEFAULT_LEVEL=2
-CONFIG_LOG_MODE_OVERFLOW=y
-CONFIG_LOG_PRINTK=y
-CONFIG_LOG_BUFFER_SIZE=4096
-CONFIG_LOG_PROCESS_THREAD_STACK_SIZE=1024
-
-CONFIG_CONSOLE=y
-CONFIG_USE_SEGGER_RTT=y
-CONFIG_SEGGER_RTT_BUFFER_SIZE_UP=4096
-CONFIG_RTT_CONSOLE=y
-CONFIG_UART_CONSOLE=n
+CONFIG_DESKTOP_LOG=y
+
+CONFIG_DESKTOP_SHELL=y
 
-CONFIG_SHELL=y
-CONFIG_SHELL_BACKEND_SERIAL=n
-CONFIG_SHELL_BACKEND_RTT=y
-CONFIG_SHELL_PROMPT_RTT="nrf_desktop:~$ "
-CONFIG_DEVICE_SHELL=y
-CONFIG_KERNEL_SHELL=y
 CONFIG_THREAD_MONITOR=y
 CONFIG_INIT_STACKS=y
 CONFIG_THREAD_NAME=y
```

Comment:
These three options were also used for shell debug purposes. Can we keep set them by default when `CONFIG_DESKTOP_SHELL` is enabled? The options were used to profile thread stack usage

Reply:
And profiling stack usages used to be one of the main reasons to use shell in nRF Desktop

Reply:
sure, it is fixed now